config/environments/production.rb

```diff
@@ -42,7 +42,8 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  # We terminate SSL before traffic gets to the gi data service elb and traffic from the elb to the service is over http. So forcing ssl will break our ELB health checks.
+  # config.force_ssl = true
```

Comment:
@annaswims I notice the last part of this is enabling Rails to "use secure cookies". Do you know if we would need to do anything different to make sure cookies have the secure attribute set? I'm not sure how to login to this app, otherwise I could check quick.